src/flag_gems/ops/resolve_conj.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+import torch
+
+from flag_gems.ops.neg import neg_func
+
+
+def resolve_conj(A: torch.Tensor):
+    logging.debug("GEMS RESOLVE_CONJ")
+    return neg_func(A.imag) if A.is_conj() else A
```

Comment:
The true branch only returns imaginary part of the complex tensor.

Reply:
Before a conjugate is materialized, it shares the original complex tensor's data. Resolving a conjugate complex probably requires you make a copy of the whole tensor in the first place then negate its imag part.

Reply:
will be fixed later，thanks～